pylossless/pipeline.py

```diff
@@ -550,7 +561,15 @@ def flag_ch_low_r(self, raw):
                                            **self.config['ch_low_r'])[1]
 
         # Edit the channel flag info structure
-        bad_ch_names = data_r_ch.ref_chan[flag_r_ch_inds].values.tolist()
+        try:
+            bad_ch_names = data_r_ch.ref_chan[flag_r_ch_inds].values.tolist()
+        except:
+            print(self.config['ch_low_r'])
+            print(data_r_ch.values.T)
+            print(data_r_ch.values.T.shape)
+            print(flag_r_ch_inds)
+            print(data_r_ch.ref_chan)
```

Comment:
Can we use logger here? I can loop back to this on another PR after merge

Reply:
Sorry, I did not realize it had put you as a reviewer! This PR was actually not be meant to be revised at this time. I made it as a PR to more easily track its progress, but I still have to add many commits to this PR before it is ready for merging!

The print statements there, for example, are not meant to remain.